public/plugins/testdata/.eslintignore

```diff
@@ -0,0 +1,2 @@
+# TS generate from cue by cuetsy
+**/*.gen.ts
```

Comment:
copied from the main .eslintignore (but necessarsy to run eslint from this folder only)